app/main.py

```
@@ -0,0 +1,30 @@
+
+app.include_router(api_v1)
+app.add_middleware(PerformanceMiddleware)
+
+
+@app.exception_handler(ClientException)
+async def client_exception_handler(request: Request, e: ClientException):
+    return ORJSONResponse(status_code=e.status_code, content={"detail": e.detail})
+
+
+@app.exception_handler(ScraperException)
+async def scraper_exception_handler(request: Request, e: ScraperException):
```

Ruff (0.11.9)

Missing return type annotation for public function `scraper_exception_handler`

(ANN201)